src/org/mirah/builtins/string_extensions.mirah

```diff
@@ -39,4 +39,27 @@ class StringExtensions
       end
     end
   end
+  
+  # separate to to_i(), as to_i() (in order to maintain Ruby compatibility) could return a BigInteger,
+  # while to_int() always returns an int.
+  macro def to_int:int
```

Comment:
I like it!
This makes me think wonder if the cast fns should also work this way--ie `int('1')` would cast the string to a primitive int, and `Integer('1')` would as well.